trillian/ctfe/config.go

```diff
@@ -231,18 +231,62 @@ func BuildLogBackendMap(lbs *configpb.LogBackendSet) (LogBackendMap, error) {
 	return lbm, nil
 }
 
+func commonValidationLogConfigs(cfg []*configpb.LogConfig) error {
```

Comment:
That name is a bit ugly. Just `validateConfigs` or similar maybe?

Reply:
Done.